packages/appeals-service-api/src/validators/appeals/appeals.validator.js

```diff
@@ -14,8 +25,22 @@ const appealUpdateValidationRules = async (req, res, next) => {
 const appealInsertValidationRules = async (req, res, next) => {
   try {
     req.body = await insertAppeal.validate(req.body, { abortEarly: false });
+    logger.debug('Valid input format');
+
+    const appealId = req.body.id;
+    const pathId = req.params.id;
+    if (await isAppealSubmitted(appealId)) {
+      logger.debug('Appeal is already submitted so end processing request with 409 response');
+      next(ApiError.appealAlreadySubmitted());
```

Comment:
same issue as above, and potentially more serious this time (?)

Reply:
Made the change